apps/api/prisma/seed-samples.js

```diff
@@ -602,45 +602,57 @@ export const regions = [
 
 export const zoomLevels = [
 	{
-		id: 127,
-		name: 'null',
-		displayNameEn: 'NULL',
-		displayNameCy: 'NULL'
-	},
-	{
-		id: 7,
-		name: 'borough',
-		displayNameEn: 'Borough',
-		displayNameCy: 'Borough'
-	},
-	{
-		id: 12,
 		name: 'country',
+		displayOrder: 900,
 		displayNameEn: 'Country',
 		displayNameCy: 'Country'
 	},
 	{
-		id: 29,
+		name: 'region',
+		displayOrder: 800,
+		displayNameEn: 'Region',
+		displayNameCy: 'Region'
+	},
+	{
 		name: 'county',
+		displayOrder: 700,
 		displayNameEn: 'County',
 		displayNameCy: 'County'
 	},
 	{
-		id: 2,
+		name: 'borough',
+		displayOrder: 600,
+		displayNameEn: 'Borough',
+		displayNameCy: 'Borough'
+	},
+	{
 		name: 'district',
+		displayOrder: 500,
 		displayNameEn: 'District',
 		displayNameCy: 'District'
 	},
 	{
-		id: 105,
-		name: 'region',
-		displayNameEn: 'Region',
-		displayNameCy: 'Region'
+		name: 'city',
+		displayOrder: 400,
+		displayNameEn: 'City',
+		displayNameCy: 'City'
 	},
 	{
-		id: 1,
 		name: 'town',
+		displayOrder: 300,
 		displayNameEn: 'Town',
 		displayNameCy: 'Town'
+	},
+	{
+		name: 'junction',
+		displayOrder: 200,
+		displayNameEn: 'Junction',
+		displayNameCy: 'Junction'
+	},
+	{
+		name: 'null',
+		displayOrder: 0,
+		displayNameEn: 'NULL',
+		displayNameCy: 'NULL'
 	}
```

Comment:
Do we need this one?

Reply:
can you change the displayName as well? in the design is None